projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration.ts

```diff
@@ -117,14 +117,25 @@ export function selectAttribute(
     case 'dropdown':
       cy.get(`#${attributeId} ng-select`).ngSelect(valueName);
       return;
-
+    case 'checkBoxList':
+      cy.get(`#${valueId}`).click({ force: true });
```

Comment:
duplicate line of code from radioGroup case.
Do instead use for both: 

case 'radioGroup':
case 'checkBoxList':
      cy.get(`#${valueId}`).click({ force: true });
      break;